common/djangoapps/student/models.py

```diff
@@ -3103,3 +3103,16 @@ def should_celebrate_first_section(enrollment):
             return enrollment.celebration.celebrate_first_section
         except CourseEnrollmentCelebration.DoesNotExist:
             return False
+
+
+class UserAccountDisableHistory(TimeStampedModel):
+    """
+    Keeps track of user disable/enable history
+    """
+    user = models.ForeignKey(User, related_name='disable_comment', on_delete=models.CASCADE)
+    comment = models.CharField(max_length=255, help_text=_("Add a reason"), blank=True, null=True)
+    disabled = models.BooleanField(default=True)
+    by = models.ForeignKey(User, related_name='users_disabled', on_delete=models.CASCADE)
```

Comment:
`by` => `disabled_by`

Reply:
@awaisdar001 This model also tracks enabled account. So that is why I kept it generic as "by"

Reply:
fair point. in that case, what about `created_by`?